contrib/kea/update.py

```diff
@@ -0,0 +1 @@
+ignore = ["*.*[13579].*"]
```

Comment:
you also have to set url= so it picks up the new 2.6 version (since it doesn't currently)

so,

```
url = "https://downloads.isc.org/isc/kea"
pattern = r">([\d\.]+)/<"
ignore = ["*.*[13579].*", "201*"]
```

Reply:
oh I missed 2.6.0 was released 2 days ago.  Good catch, thanks!  I will work on adding 2.6.0 instead